cas/service.c

```diff
@@ -1362,8 +1438,8 @@ static int cas_fom_tick(struct m0_fom *fom0)
 			addb2_add_kv_attrs(fom, STATS_KV_OUT);
 		} else {
 			do_ctidx = cas_ctidx_op_needed(fom, opc, ct, ipos);
-			result = cas_exec(fom, opc, ct, ctg, ipos,
-					  is_index_drop ?
+			result = cas_exec(fom, opc, ct, ctg, ipos, is_index_drop
+					?
```

Comment:
Why do you have this change?

According to coding-style, we don't put operator (here is the ?) at the beginning of a line.

Reply:
No need, returned back.